src/domain/gpqueue.ts

```diff
@@ -55,6 +55,8 @@ export function create(dao: Address,
        gpAddress = parameters.value1;
        setContributionRewardParams(dao, scheme, gpAddress, parameters.value0);
        isGPQue = true;
+     } else {
+       setSchemeParamsError(dao, scheme);
```

Comment:
it might be cleaner to get the schemeId earlier .
use it also for 
https://github.com/daostack/subgraph/blob/11e36b95e14f66867f6b1290d39f88b006ec79c5/src/domain/gpqueue.ts#L125
and call setSchemeParamsError with schemeId